assets/views/layout.ejs

```diff
@@ -25,7 +25,13 @@
     </script>
   </head>
 
-  <body ng-controller="userCtrl">
+  <body ng-controller="userCtrl" ng-cloak ng-init="init({<%=
+    (typeof user !== 'undefined' ? 'user:' + JSON.stringify(user) + ',' : '') +
+    (typeof refUser !== 'undefined' ? 'refUser:' + JSON.stringify(refUser) + ',' : '') +
+    (typeof flairs !== 'undefined' ? 'flairs:' + JSON.stringify(flairs) + ',' : '') +
+    (typeof flairApps !== 'undefined' ? 'flairApps:' + JSON.stringify(flairApps) + ',' : '') +
+    (typeof query !== 'undefined' ? 'query:' + JSON.stringify(query) + ',' : '')
+  %>})">
```

Comment:
I do hate this, but it makes other things nicer, and I don't think there's a nicer way to do it. So that's fine.